src/js/disability-benefits/actions/index.js

```diff
@@ -68,7 +68,13 @@ export function getClaimDetail(id) {
       null,
       dispatch,
       resp => dispatch({ type: SET_CLAIM_DETAIL, claim: resp.data, meta: resp.meta }),
-      () => dispatch({ type: SET_UNAVAILABLE })
+      resp => {
+        if (resp.status !== 404 || !router) {
+          dispatch({ type: SET_UNAVAILABLE });
+        } else {
+          router.replace('your-claims');
```

Comment:
It looks like this could be unit tested with a sinon spy/stub as `router` is passed in to this method